openff/toolkit/topology/molecule.py

```diff
@@ -1163,6 +1170,27 @@ def to_dict(self) -> dict:
         molecule_dict
             A dictionary representation of the molecule.
 
+            - **name** (str): An optional name to be associated with the molecule
+            - **atoms** (list[dict]): A list of dictionary inputs for :meth:`Atom.from_dict()`
+            - **bonds** (list[dict]): A list of dictionary inputs for :meth:`Bond.from_dict()`
+            - **conformers** (list[list]): A list containing the cartesian coordinates of each atoms in the order
+              defined in ``atoms``.
+            - **properties** (dict): Outputs from chosen a chosen toolkit:
+
+                - **atom_map** (dict): Dictionary of atom index (as in ``atoms`` entry) and the mapped index relevant
+                  to a mapped canonical smiles string
+                - **\*\*kwargs**: Other toolkit dependent outputs
+
+            - **hierarchy_schemes** (dict[dict]): Dictionary where keys (such as ``"residues"`` and ``"chains"``)
+              represent dictionary outputs from :meth:`HierarchyScheme.to_dict()`
+            - **conformers_unit** (float, default="angstrom"): Valid unit of length input for the
+              `OpenFF Units module <https://docs.openforcefield.org/projects/units/en/stable/api/generated/openff.units.html>`_.
```

Comment:
Incorrect type here

```suggestion
            - **conformers_unit** (str, default="angstrom"): Valid unit of length input for the
              `OpenFF Units module <https://docs.openforcefield.org/projects/units/en/stable/api/generated/openff.units.html>`_.
```

Strictly speaking this appears to be an optional key? I was surprised before checking

```python
In [11]: Molecule.from_smiles("CCO").to_dict().keys()
Out[11]: dict_keys(['name', 'atoms', 'bonds', 'properties', 'conformers', 'partial_charges', 'partial_charge_unit', 'hierarchy_schemes'])